docs/graphql/manual/deployment/heroku/updating.rst

```diff
@@ -50,6 +50,8 @@ When you ``git push`` to deploy, the Heroku app will get updated with the latest
 
    $ git push heroku master
 
+If you are getting error: ``fatal: 'heroku' does not appear to be a git repository``, you need to add your heroku repo to your local git remote via: ``git remote add heroku https://git.heroku.com/<you-app-name>.git``, you can find this URL in your Heroku - Settings - Info - Heroku Git URL. Type `git remote -v`, you will see your `heroku` app has been added as the upstream.
```

Comment:
Can we write: "If you are getting _the following_ error: ..."? 

And could we make two or three sentences here instead of one long one? I think this would help for the understanding 🙂

Reply:
Sounds good, thanks for reviewing! updated!